Tensile/KernelWriterAssembly.py

```diff
@@ -318,7 +318,7 @@ def __init__( self, kernelMinNaming, kernelSerialNaming ):
     # 0x8 = waitcnt at self.wait() for localRead
     self.db["ConservativeWaitCnt"] = 0x0
 
-    self.db["InitLds"]     = False  # Initialize LDS at start of kernel
+    self.db["InitLds"]     = True  # Initialize LDS at start of kernel
```

Comment:
We sure we want to turn this on by default? That would mean a perf hit unless people disable InitLds in the YAML.